lib/rules/disallow-keywords-in-comments.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Disallows keywords in your comments, such as TODO or FIXME
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * #### Example
+ *
+ * `"disallowKeywordsInComments": true`
+ *
+ * Invalid:
+ * ```
+ * // ToDo
+ * //TODO
+ * /** fixme *\/
+ * /**
+ *  * FIXME
+ *  *\/
+ * ```
+ */
+
+var assert = require('assert');
+
+var r = /\b(todo|fixme)\b/i;
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(disallowKeywordsInComments) {
+        assert(
+            disallowKeywordsInComments === true,
+            'disallowKeywordsInComments option disallows certain keywords in your comments'
+        );
+    },
+
+    getOptionName: function() {
+        return 'disallowKeywordsInComments';
+    },
+
+    check: function(file, errors) {
+        file.getComments().forEach(function(comment) {
+            var errorLoc = {};
+            // split by new line to get accurate line number reporting
+            var splitComment = comment.value.split('\n');
+
+            splitComment.forEach(function(commentNode, index) {
+                var matchIndex = commentNode.search(r);
```

Comment:
What if the word TODO appears twice in the same line? You'll only report one error...

Which I think is fine, but it highlights the fact that getting the exact location of the offending word is probably not important. You could simplify this rule greatly if you just reported the start of the comment as the error location.